src/pages/NewListing.jsx

```diff
@@ -419,17 +433,45 @@ const NewListing = () => {
           }
         </FlexRow>
         <FlexRow>
-        {<CheckCircle color="success" />}
-          <div className="zoom-burn-fee">
-            Zoom <StyledLogo src={zoomLogo} /> Burn Fee:{' '}
-            {data && data.zoomBurnFee
-              ? data.zoomBurnFee * numberOfSelectedCards
-              : 0}{' '}
-            {' '}
-          </div>
+          <FlexColumn>
+            <FlexRow>
+              {
+                !exceedZoomAllowance &&
+                (
+                  <CheckCircle color="success" />
+                )
+              }
+              <div className="zoom-burn-fee">
+                Zoom <StyledLogo src={zoomLogo} /> Burn Fee:{' '}
+                {data && data.zoomBurnFee
+                  ? data.zoomBurnFee * numberOfSelectedCards
+                  : 0}{' '}
+                { currentAllowance !== undefined ? '(Allowance : ' + currentAllowance + ')' : ''}
```

Comment:
nit, can you use string literal instead of `+` concat?

Reply:
Sure.